app/src/main/kotlin/com/yubico/yubioath/ui/main/CredentialFragment.kt

```diff
@@ -172,7 +172,11 @@ class CredentialFragment : ListFragment(), CoroutineScope {
 
         btn_scan_qr.setOnClickListener {
             hideAddToolbar()
-            startActivityForResult(Intent(activity, QrActivity::class.java), REQUEST_SCAN_QR)
+            if (activity?.isGooglePlayAvailable() == true) {
```

Comment:
nit: you can omit == true

Reply:
Not in this case I am afraid, `activity` is nullable so the compiler forces me to do this treatment. I could surround it with a `let` block, etc. but I think this is the cleaner approach. Do you have any suggestion?

Reply:
You need context, not Activity, and on click listener has view argument. That's where you can get nonnull context

Reply:
Right! That is definitely a better approach!